ui/package.json

```diff
@@ -10,11 +10,9 @@
     "dagre": "^0.8.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-flow-renderer": "^10.2.1",
+    "react-flow-renderer": "^9.7.4",
```

Comment:
why the downgrade?

Reply:
Some code works in v9 have issues in v10. Indented to use v9 for now as v9 is more popular, download volume v9 latest is 60k while v10 is a few hundreds.